block/test/Test/Pos/Block/Arbitrary.hs

```diff
@@ -29,7 +29,7 @@ import           Pos.Arbitrary.Update (genUpdatePayload)
 import           Pos.Binary.Class (biSize)
 import qualified Pos.Block.Base as T
 import qualified Pos.Block.Logic.Integrity as T
-import           Pos.Block.Slog.Types (SlogUndo)
+import           Pos.Block.Slog (SlogUndo)
```

Comment:
The module `Pos.Block.Slog.Types` is not exposed by the package `block`, but it is re-exported by the module `Pos.Block.Slog` (therefore giving me access from `block-test`).